src/Umbraco.Core/Services/IContentTypeServiceBase.cs

```diff
@@ -78,7 +78,7 @@ public interface IContentTypeBaseService<TItem> : IContentTypeBaseService, IServ
         /// <returns></returns>
         bool HasContainerInPath(params int[] ids);
 
-        Attempt<OperationResult<OperationResultType, EntityContainer>> CreateContainer(int parentContainerId, string name, int userId = Constants.Security.SuperUserId);
+        Attempt<OperationResult<OperationResultType, EntityContainer>> CreateContainer(int parentContainerId, Guid key, string name, int userId = Constants.Security.SuperUserId);
```

Comment:
@bergmania please flag that pr as breaking, we are doing changes in to public interfaces :)